Playbooks/Get-GeoFromIpAndTagIncident.json

```diff
@@ -1,6 +1,10 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
+    "metadata":{
+        "comments": "This playbook will take the IP address entities from the Incident and query a Geo-IP API to geo locate the IP Address.  It will write the City and Country to a tag on the Incident.",
```

Comment:
I think this should be "geo-locate".  Otherwise, I think you have to state "geographically locate".